### PR TITLE
SceneView: lighting options microapp

### PR DESCRIPTION
**Pull request by hud10837**
Adds a microapp to demonstrate the various lighting options in the SceneView:

- Sun Time
- Sun Lighting
- Ambient Light Color
- Atmosphere Effect
- Space Effect

### CODE REVIEW

File: microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/DropdownMenuAlertDialog.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+@Composable
+fun DropdownMenuAlertDialog(
+    itemList: List<String>,
+    currentSelectedIndex: Int,
+    title: String,
+    onDismissRequest: () -> Unit,
+    onConfirm: (Int) -> Unit
+) {
+    var dropdownExpanded by remember { mutableStateOf(false) }
+    var selectedIndex by remember { mutableIntStateOf(currentSelectedIndex) }
```
Comment (changanxian) [inline microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/DropdownMenuAlertDialog.kt:42]: Locally I cannot compile the class, it misses the imports,
```
import androidx.compose.runtime.getValue
import androidx.compose.runtime.setValue

```

File: microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/DropdownMenuAlertDialog.kt [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+ *
+ * The index of the selected item when the user presses the confirm button is passed to [onConfirm].
+ */
+@Composable
+fun DropdownMenuAlertDialog(
+    itemList: List<String>,
+    currentSelectedIndex: Int,
+    title: String,
+    onDismissRequest: () -> Unit,
+    onConfirm: (Int) -> Unit
```
Comment (changanxian) [inline microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/DropdownMenuAlertDialog.kt:39]: Could you add the KDoc for the parameters?

File: microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/DropdownMenuAlertDialog.kt [hunk 3]
```diff
@@ -0,0 +1,96 @@
+package com.arcgismaps.toolkit.sceneviewlightingoptionsapp.screens
```
Comment (changanxian) [inline microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/DropdownMenuAlertDialog.kt:1]: missing License header

File: microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/RgbaSlider.kt [hunk 4]
```diff
@@ -0,0 +1,37 @@
+package com.arcgismaps.toolkit.sceneviewlightingoptionsapp.screens
```
Comment (changanxian) [inline microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/RgbaSlider.kt:1]: missing copyright

File: microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/RgbaSlider.kt [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+
+/**
+ * Displays a slider with a range of 0 to 255, set to the initial [value].
+ */
```
Comment (changanxian) [inline microapps/SceneViewLightingOptionsApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewlightingoptionsapp/screens/RgbaSlider.kt:15]: better to add the KDoc for the parameters

**Comment by changanxian** [comment]
@hud10837 looks good, a few comments

**Comment by hud10837** [comment]
Thanks @changanxian I have added all the remaining doc

**Review by changanxian — APPROVED** [review]
👍

File: microapps/SceneViewLightingOptionsApp/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# SceneView Lighting Options Micro-app
+
+This micro-app demonstrates the the use of various lighting and rendering options for the composable `SceneView`
```
Comment (gunt0001) [inline microapps/SceneViewLightingOptionsApp/README.md:3]:
```suggestion
This micro-app demonstrates the use of various lighting and rendering options for the composable `SceneView`
```

**Review by gunt0001 — APPROVED** [review]
@hud10837 - looks great, one minor comment.

**Pull request merged**